cms/djangoapps/contentstore/views/tests/test_library.py

```diff
@@ -324,7 +324,7 @@ def test_manage_library_users(self):
         response = self.client.get(manage_users_url)
         self.assertEqual(response.status_code, 200)
         # extra_user has not been assigned to the library so should not show up in the list:
-        self.assertNotIn(extra_user.username, response.content)
+        self.assertNotIn(str(extra_user.username), response.content)
```

Comment:
I don't think this will work for Python 3; maybe use `six.binary_type(extra_user.username)` instead?

Reply:
Cleaned up that and the uses of unicode